build/linux/Dockerfile

```diff
@@ -1,8 +1,31 @@
+ARG BASE_IMAGE=portainer/base
 ARG GIT_COMMIT=unspecified
-FROM portainer/base:latest
+ARG GO_VERSION=1.22.7
```

Comment:
Whilst this is the latest patch version for 1.22, I believe this is not aligned with other software which is built upon 1.22.6.

I'd standardize over a single version and make sure that you have a tracking sheet / doc somewhere that tells you where this version is defined so that you can easily update it in the future.

Reply:
In order to fix some of the recent "High" CVE 1.22.7 bump was done for 2.22.x(=develop) about 2 weeks ago.
https://linear.app/portainer/issue/BE-11188/2024-september-cve (34158 etc.)

I added that ARG GO_VERSION=1.22.7 with that in mind, just as a default. I will drop it to 1.22.6 as the "default"
Overall, my intention was to override that as a build arg and bump the version from outside the Dockerfile.

Reply:
All good I missed that during my holidays. I'll let you manage the Go version but make sure it is consistent.

Reply:
I used the [ticket/thread](https://linear.app/portainer/issue/PLA-76/inconsistent-go-language-versions-used-in-different-repositories#comment-c9ed577c) at the time we bumped GO_VERSION to 1.22.6 (and removed that toolchain directive from go.mod) and updated the go.mod in each repository followed by running `go mod tidy`

- https://github.com/portainer/portainer
- https://github.com/portainer/portainer-ee
- https://github.com/portainer/agent
- https://github.com/portainer/compose-unpacker
- https://github.com/portainer/portainer-updater
- https://github.com/portainer/license-server
- https://github.com/portainer/api-gateway
- https://github.com/portainer/helper-reset-password